src/ProjectWindow.cpp

```diff
@@ -1266,9 +1256,9 @@ bool ProjectWindow::IsIconized() const
    return mIconized;
 }
 
-wxWindow* ProjectWindow::GetEffectsWindow() noexcept
+RealtimeEffectPanel &ProjectWindow::GetEffectsWindow() noexcept
```

Comment:
We disagree about `noexcept`.  I notice you add it often.  I do not.

I'm not very certain what is right for `noexcept`.  I have heard arguments in Cppcon talks that it is not right to add it everywhere you can, just because you can.  It is meant for more specific uses, and should be used only when that is conveying some such related intent.

`noexcept` can be of use inside templates that text the conditional noexcept expressions, as in `if constexpr`, to compile more efficient versions of containers that don't need to implement strong exception safety guarantees.  But this is nothing like that.

Myself, I add `noexcept` when a virtual function is meant to be used inside a destructor, therefore its overrides must definitely not throw.  This is a way of documenting subclass responsibility.

I do not use it much elsewhere.

Reply:
Anyway, as you know now from completing the review, this is only a line in a transitional commit, before eliminating this function.

Were you saying that in the transitional commit, a `noexcept` function was calling another that was not `noexcept`?  True.

But then my opinion is that the unnecessary `noexcept` on the calling function should not have been there.